
public class Ex4 {

	public static void main(String[] args) {
		// TODO Auto-generated method stub

	}



public static void foo(int x,int y) {
	
try {
int[] arr = {1,2,3};
}catch(ArithmeticException e) {
	System.out.println("Error");
}catch(ArrayIndexOutOfBoundsException e) {//can also be IndexoutfBoundsException
	System.out.println("Out of bound exception");
}catch(Exception e) {
	System.out.println("Erroe!");
}
	
	



}





}
